admin/class-acc_user_importer-admin.php

```diff
@@ -404,10 +430,12 @@ private function proccess_user_data ( $users ) {
 
 			//--------CREATE NEW USER-----
 			$this->log_dual(" > email not found on any other users");
-			$this->log_dual(" > will create new user account");
 			$new_users[] = $userContactId;
 			$new_users_email[] = $userEmail ;
 			$accUserData["role"] = $default_role;
+			$accUserData["user_registered"] = date('Y-m-d H:i:s');
```

Comment:
This is done automatically by WordPress: https://github.com/WordPress/wordpress-develop/blob/5.8.1/src/wp-includes/user.php#L1904

I think we can remove this again.